packages/preview/ccicons/1.0.0/src/ccicons.typ

```diff
@@ -0,0 +1,53 @@
+#import "utils.typ": ccicon, cc-url, cc-is-valid, _cc-colorize
+
+// Logos
+#let ccLogo = ccicon("logo")
```

Comment:
Typst packages use kebab-case and not camelCase by convention (and you seem to use that in other parts of your package as well). Following this is not mandatory to get your package accepted though, so if you prefer to keep camelCase names, just tell me.

Reply:
I kept the same function names as in the original package. I think I would like to keep the current naming scheme. 

The individual icons will probably not get much use, anyway. Most people will use the license icons like `#cc-by-nc-sa` and those are in the usual format.